Add code of conduct section and page

deweydell opened the pull request:
- add to nav
- add html and styles
- add .rounded-corners utility class

deweydell committed:
Add code of conduct section and page

- add to navbar
- add html and styles
- add .rounded-corners utility class

jarmstrng commented on index.html:
```diff
@@ -316,12 +316,14 @@ <h2 class="section-title">Venue</h2>
+				</div>
+				<div class="col-xs-12 col-sm-6 section-button-cta">
+					<a href="assets/sponsor-prospectus.pdf" target="_blank" class="button secondary">Interested in becoming a sponsor?</a>
+				</div>
```
ooh, what's this change?

deweydell replied:
In the original design, the grid system hides the the column on the right on smaller screens. Here I changed the columns here to the flexboxgrid classes so that the "Interested in becoming a sponsor" button was visible at all times since I thought that was easier than changing the theme css.

jarmstrng replied:
👍

jarmstrng commented on index.html:
```diff
@@ -348,6 +350,32 @@ <h2 class="section-title">Sponsors</h2>
+			<div class="row">
+				<div class="col-xs-12 col-sm-7">
+					<h3>The Quick Version</h3>
+					<p>Our event and its associated online spaces are dedicated to providing a harassment-free experience for everyone, regardless of gender and gender identity, age, sexual orientation, disability, physical appearance, body size, race, or religion (or lack thereof). We do not tolerate harassment of participants in any form. Sexual language and imagery is not appropriate at any point during the  ...
```
Can we get the line height here & in the Need Help? section to match what we have in, say, the sponsor blurb?

deweydell replied:
👍

jarmstrng commented on index.html:
```diff
@@ -316,12 +316,14 @@ <h2 class="section-title">Venue</h2>
 <!-- ## SPONSOR SECTION ## -->
 	<section class="sponsors-section qcContainer" id="sponsors">
 	  <div class="qcTabTitle clearfix">
```
is it just me or does the sponsor section have slightly more whitespace above the headline?

deweydell replied:
ok I'll look into it

jarmstrng commented on code-of-conduct.html:
```diff
@@ -0,0 +1,93 @@
+
+  <footer>
+    2017 &copy; Write/Speak/Code
+  </footer>
```
change to `&copy; 2017 Write/Speak/Code`

jarmstrng commented on code-of-conduct.html:
```diff
@@ -0,0 +1,93 @@
+            <li>Deliberate misgendering or use of ‘dead’ or rejected names.</li>
+            <li>Gratuitous or off-topic sexual images or behaviour in spaces where they’re not appropriate.</li>
+            <li>Physical contact and simulated physical contact (eg, textual descriptions like “*hug*” or “*backrub*”) without consent or after a request to stop.</li>
+            <li>Threats of violence.</li>
```
remove asterixes (asterixi?) from "*hug*" and backrub

jarmstrng commented on code-of-conduct.html:
```diff
@@ -0,0 +1,93 @@
+            <li>Reasonable communication of boundaries, such as “leave me alone,” “go away,” or “I’m not discussing this with you.”</li>
+            <li>Communicating in a ‘tone’ you don’t find congenial</li>
+            <li>Criticizing racist, sexist, cissexist, or otherwise oppressive behavior or assumptions</li>
+          </ul>
```
add periods to the ends of these bullet points to be consistent with other section

jarmstrng commented on code-of-conduct.html:
```diff
@@ -0,0 +1,93 @@
+          </ul>
+
+          <h4>Write/Speak/Code prioritizes marginalized people’s safety over privileged people’s comfort and therefore we will not act on complaints regarding:</h4>
+          <ul>
```
the  `<h4>` section here is a bit hard to read, maybe remove the capitalization or change the line height or... whatever looks good

jarmstrng commented on code-of-conduct.html:
```diff
@@ -0,0 +1,93 @@
+        </a>
+        <h1>Conference Code of Conduct</h1>
+        <a href="index.html" target="_blank" class="button secondary">back to conference website</a>
+      </div>
```
i like the CoC opening in a new tab but having the conf website open in a new tab from here = lots of tabs. what do you think?

deweydell replied:
good call 👍

jarmstrng reviewed (dismissed):
looks good on mobile

jarmstrng commented:
oh yeah and ping me once it's on staging 😁

deweydell committed:
Add change requests from pr review

- Adopt similar linespacing in blurbs
- Remove target blank from conference button on coc page
- Remove Asterisks from coc page
- Add periods to list items for consistency
- Wrap .dont-break-out class in media query so it doesn’t break links
on larger screens
- Remove extra margin from sponsor section

deweydell committed:
Tweak padding, fonts, colors
